web/viewer.html

```diff
@@ -644,7 +666,7 @@
               <button id="altTextSettingsCloseButton" type="button" class="primaryButton" tabindex="0"><span data-l10n-id="pdfjs-editor-alt-text-settings-close-button">Close</span></button>
             </div>
           </div>
-        </dialog>
+        </dialog>        
```

Comment:
Please remove the trailing spaces.
```suggestion
        </dialog>
```